tests/test_tasty.py

```diff
@@ -0,0 +1,57 @@
+from tests import ScraperTest
+
+from recipe_scrapers.tasty import Tasty
+
+
+class TestTastyScraper(ScraperTest):
+
+    scraper_class = Tasty
+
+    def test_host(self):
+        self.assertEqual("tasty.co", self.harvester_class.host())
+
+    def test_title(self):
+        self.assertEqual(
+            self.harvester_class.title(),
+            "Red Wine-Braised Short Ribs With Cashew Cauliflower Mash Recipe by Tasty",
+        )
+
+    def test_yields(self):
+        self.assertEqual("4 servings", self.harvester_class.yields())
+
+    def test_image(self):
+        self.assertEqual(
+            "https://img.buzzfeed.com/thumbnailer-prod-us-east-1/video-api/assets/236025.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_ingredients(self):
+        self.assertCountEqual(
+            [
+                "3 lb bone in beef short ribs, cut into 2-inch pieces",
+                "1 tablespoon kosher salt",
+                "2 teaspoons kosher salt",
+                "1 ÇÂ teaspoons freshly ground black pepper",
```

Comment:
If you run `coverage run -m unittest` you should see that there is an error with this line as well as line 48.